src/emulator/app/src/app_init.cpp

```diff
@@ -174,6 +178,12 @@ bool init(HostState &state, Config cfg, const Root &root_paths) {
         return false;
     }
 
+#ifdef USE_DISCORD_RICH_PRESENCE
+    discord::init();
+    if (cfg.discord_rich_presence) {
+        discord::update_presence("");
```

Comment:
```suggestion
        discord::update_presence();
```
you already have a default argument
but whatever it's fine